lib/msf/core/exploit/remote/smb/log_adapter.rb

```diff
@@ -0,0 +1,90 @@
+# -*- coding: binary -*-
+
+module Msf::Exploit::Remote::SMB::LogAdapter
```

Comment:
Not a blocker: In a turn of events, I'd like this functionality over in Kerberos-land as well 😄